.github/workflows/ci.yml

```diff
@@ -7,23 +7,53 @@ jobs:
     name: Windows unit tests
     runs-on: windows-latest
     steps:
+      - uses: actions/cache@v1
+        with:
+          path: ${%LOCALAPPDATA%\Coursier\Cache\v1}
+          key: ${{ runner.os }}-coursier-ci
+          restore-keys: |
+            ${{ runner.os }}-coursier-
       - uses: actions/checkout@v1
       - uses: olafurpg/setup-scala@v5
       - name: Run unit tests
         run: bin/test.sh "unit/test"
```

Comment:
Can we actually have a build matrix with different commands to run? There seems to be more and more repeated code.

Reply:
I'll work on it when I manage to get cache to work.

Reply:
I think it's better to generate `ci.yml` from build.sbt, I experimented with that approach here https://github.com/scalameta/metals/pull/940

Reply:
Sbt seems like an overkill. It can easily be done via script or I think it would be much better to solve it using the actions syntax. Why would using `sbt` be neccessary? We do not use any information from the build itself.

Reply:
Using sbt would allow us to generate a custom `cross/test` job for `crossScalaVersions`